app/views/user_mailer/in_person_please_call.html.erb

```diff
@@ -8,7 +8,7 @@
 <p>
   <%= t(
         'user_mailer.in_person_please_call.body.intro_html',
-        date: I18n.l(14.days.from_now, format: I18n.t('time.formats.event_date')),
+        date: I18n.l(14.days.from_now, format: locale == :zh ? I18n.t('time.formats.full_date') : I18n.t('time.formats.event_date')),
```

Comment:
Wanted to rephrase my [same question from the previous PR](https://github.com/18F/identity-idp/pull/10575#discussion_r1596031697):

How come we are changing formats for just one locale? In all instances of `event_date` do we want to use a `full_date` for `zh` locale? Or just this one instance? 

It seems very error-prone to me to have this kind of difference, and I'd expect either:
- update `event_date` for `zh` to match this situation
- if something is more situational, define a new date format, that has these properties for the right locales?